### ISSUE TITLE
no tag values returned

### ISSUE THREAD

**Issue by ikovalyov** [influxdata/influxdb#9529]
### Bug report

__System info:__ 1.4.3

__Steps to reproduce:__

1. I have db with country tag.

__Expected behavior:__ show tag values should return tags

__Actual behavior:__ empty result

```
> select * FROM two_days_cdn.cdndata limit 10
name: cdndata
time                cdnid country ms up
----                ----- ------- -- --
1520337646115007500 1     UA      32 1
1520337646115007500 2     UA      31 1
1520337646115007500 3     UA      31 1
1520337646200423600 4     UA      24 1
1520337646200423600 6     UA      26 1
1520337646200423600 7     UA      26 1
1520337646335545700 13    UA      62 1
1520337646335545700 8     UA      31 1
1520337646335545700 9     UA      25 1
1520337646487353900 14    UA      40 1
```
```
> SHOW TAG VALUES ON "cdnperf" FROM two_days_cdn.cdndata WITH KEY IN ("country")
>
```
```
> show tag keys from cdnperf.two_days_cdn.cdndata
>
```
```
> SHOW FIELD KEYS from cdnperf.two_days_cdn.cdndata
name: cdndata
fieldKey fieldType
-------- ---------
ms       float
up       float
```

Somehow cdnid and country are neither fields nor tags.

**Comment by e-dard** [comment]
@ikovalyov can you upgrade to `1.5.0` and see if you still have the same issue? Do you have any test data to reproduce on `1.4.3`?

**Comment by ikovalyov** [comment]
New db version didn't fix this issue. Also have no idea how to reproduce it :(

**Comment by codylewandowski** [comment]
@ikovalyov Did you ever find a cause or resolution to this? We're suddenly seeing this same issue today on a DB we've had running for close to a year.

**Comment by codylewandowski** [comment]
I think we found our issue. It appears that last night when the oldest shard was being deleted, the DB must not have succeeded in fully removing it. We could see through `show shards` that the oldest shard should not have still been there. Manually dropping that shard resolved the issue and all of our `show` commands worked again. We took a backup before dropping, so if we have some time, we'll try digging in a little more.
